docs/Project.toml

```
@@ -8,7 +8,7 @@ Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 
 [compat]
 Distances = "0.10"
-Documenter = "0.27"
+Documenter = "0.27, 1"
```

I think we should only support
```suggestion
Documenter = "1"
```
since we remove `strict=true`.